### DISCUSSION TITLE
Choose new logo

### DISCUSSION THREAD

**Discussion by sadellie** [sadellie/unitto#26] [Polls]
![poll](https://user-images.githubusercontent.com/10384356/222930265-315ec219-5c2f-4bc1-9034-8b303ef14ddd.png)

- _Logos here are not in their final version._
- You can suggest your logo ideas. (No generic calculator icons, please)

# POLL CLOSED
_**Thank you**_ for participating. Results are below.

## Launcher icon (the one you see on your home screen): 
I decided to add an option that will let you choose a logo (all 7 options) and color. (By default Unitto will use store icon)

## Store icon (the one you see in app stores):
I will probably choose between option **1** and **5**.

## When?
Soon™.

## Final results:
| Icon | Votes % |
| --- | --- |
| **1** | 28% |
| **2** | 0% |
| **3** | 0% |
| **4** | 7% |
| **5** | 35% |
| **6** | 7% |
| **7** | 21% |

ref: #5 #25

**Comment by surendrajat** [comment]
Given that the main thing is calculator, none of them really suggest that. I hope you can update it a bit to indicate calculator(maybe like typical + – signs and all).
It's just my opinion and these logos do look good, just not really for a calculator app.

**Reply by ghost** [reply]
Well, you're quite right. I was also thinking the same. If this is a supposed to be _calculator_, the + - ×÷ signs should be there.
As far as my suggestion guess, I think if they choose a 2-box icon, they should add the + - icons only. If they go for a 4-box icon, all the signs may be added.

**Comment by sadellie** [comment]
Already working on implementing: https://github.com/sadellie/unitto/discussions/37
